Dockerfile

```diff
@@ -0,0 +1,85 @@
+ARG ELIXIR_VERSION=1.15.0
```

Comment:
I think the default image should be CUDA with XLA_ENV=cuda120 for performance reasons